state/utils/tls.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2021 The KEDA Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+
+	"github.com/youmark/pkcs8"
```

Comment:
Please use the x509 package from the standard library to parse PKCS#8 keys rather than importing an external security-related package: https://pkg.go.dev/crypto/x509#ParsePKCS8PrivateKey